packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -164,6 +164,15 @@ export class Blockchain {
     return await this.client.call('getrawmempool', [verbose], 'bignumber')
   }
 
+  /**
+   *
+   * @param {string} txId the transaction id
```

Comment:
Add description aligned with https://github.com/DeFiCh/ain/blob/63452763fef6d4dc7adf9df8e371def8ecd80caf/src/rpc/blockchain.cpp#L711

```suggestion
   * Get mempool data for the given transaction
   * @param {string} txId the transaction id
```